controller/env/broker.go

```diff
@@ -44,382 +36,103 @@ const (
 	ApiSessionRemovedType   = int32(edge_ctrl_pb.ContentType_ApiSessionRemovedType)
 	ApiSessionAddedType     = int32(edge_ctrl_pb.ContentType_ApiSessionAddedType)
 	ApiSessionUpdatedType   = int32(edge_ctrl_pb.ContentType_ApiSessionUpdatedType)
+	RequestClientReSyncType = int32(edge_ctrl_pb.ContentType_RequestClientReSyncType)
```

Comment:
New message allowing edge routers to request a resync. Used in error situations that can be detected during a sync.